### PR TITLE
[6.0] Replacing getDbo() with call to container

### PR DESCRIPTION
**Pull request by Hackwar**
### Summary of Changes
The call to `Factory::getDbo()` is deprecated. This PR replaces that with calls to the DI container.

### Testing Instructions
Codereview

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [X] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [X] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/components/com_actionlogs/src/Helper/ActionlogsHelper.php [hunk 1]
```diff
@@ -275,7 +276,7 @@ public static function getContentTypeLink($component, $contentType, $id, $urlVar
     public static function loadActionLogPluginsLanguage()
     {
         $lang = Factory::getLanguage();
-        $db   = Factory::getDbo();
+        $db   =  Factory::getContainer()->get(DatabaseInterface::class);
```
Comment (Quy) [inline administrator/components/com_actionlogs/src/Helper/ActionlogsHelper.php]:
```suggestion
        $db   = Factory::getContainer()->get(DatabaseInterface::class);
```
Repeat for the other instances.

**Comment by richard67** [comment]
@Hackwar I think unit tests are failing at places where the isMariaDb method of the MySQL drivers are used because this method is not part of the database interface.

**Comment by HLeithner** [comment]
As already discussed, this is not a solution it only fixes the syndrome (Deprecation warning). So for now I rebase this for 6.0 and hope that we don't need this pr and have a proper solution for all situations.

**Comment by laoneo** [comment]
I'm closing this as the target is to remove the static calls at all.

**Pull request closed without merging**